.travis.yml

```diff
@@ -1,4 +1,41 @@
 language: objective-c
-xcode_project: VelocidiSDK.xcodeproj # path to your xcodeproj folder
+os: osx
+xcode_workspace: VelocidiSDK.xcworkspace # path to your xcodeproj folder
 xcode_scheme: VelocidiSDKTests
-xcode_destination: platform=iOS Simulator,OS=10.1,name=iPad Pro (9.7-inch)
+
+script:
+  - xcodebuild -workspace VelocidiSDK.xcworkspace -scheme VelocidiSDKTests clean build test -sdk "$TEST_SDK" -destination "$TEST_DESTINATION" GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=YES GCC_GENERATE_TEST_COVERAGE_FILES=YES
+
+after_success:
+ - bash <(curl -s https://codecov.io/bash)
+
+jobs:
+  include:
+    - env: 
+        - TEST_SDK=iphonesimulator11.0 TEST_DESTINATION='platform=iOS Simulator,OS=11.0,name=iPhone X'
+      osx_image: xcode9
+    - env: 
+        - TEST_SDK=iphonesimulator11.1 TEST_DESTINATION='platform=iOS Simulator,OS=11.1,name=iPhone X'
+      osx_image: xcode9.1
+    - env: 
+        - TEST_SDK=iphonesimulator11.2 TEST_DESTINATION='platform=iOS Simulator,OS=11.2,name=iPhone X'
+      osx_image: xcode9.2
+    - env: 
+        - TEST_SDK=iphonesimulator11.3 TEST_DESTINATION='platform=iOS Simulator,OS=11.3,name=iPhone X'
+      osx_image: xcode9.3
+    - env: 
+        - TEST_SDK=iphonesimulator11.4 TEST_DESTINATION='platform=iOS Simulator,OS=11.4,name=iPhone X'
+      osx_image: xcode9.4.1
+    - env: 
+        - TEST_SDK=iphonesimulator12.0 TEST_DESTINATION='platform=iOS Simulator,OS=12.0,name=iPhone X'
+      osx_image: xcode10
+    - env: 
+        - TEST_SDK=iphonesimulator12.1 TEST_DESTINATION='platform=iOS Simulator,OS=12.1,name=iPhone X'
+      osx_image: xcode10.1
+    - env: 
+        - TEST_SDK=iphonesimulator12.2 TEST_DESTINATION='platform=iOS Simulator,OS=12.2,name=iPhone X'
+      osx_image: xcode10.2
+    - name: Cocoapods Lint
```

Comment:
Can you make this job the first one? It seems more logical to test it before all others.

Reply:
Which one? Cocoapods or testing against SDK 12.2?

Reply:
Cocoapods

Reply:
I don't agree, I think that testing the publish method should come at the end. From a development standpoint, you first test your project, only after it passes all tests successfully will you consider publishing it.

Reply:
I thought this step would lint the code, but [I was wrong](https://guides.cocoapods.org/terminal/commands.html#pod_lib_lint). Leave it as is.